content/blog/2022/07/verify-signatures-with-gha-and-prefix.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Verify image signatures with GithubActions and KeylessPrefix 
+authors:
+- Raul Cabello Martin 
+date: 2022-07-14
+---
+
+With the latest Kubewarden release v1.1.0 and the latest release of
+the [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy, it's now possible to use
+`GithubActions` or `KeylessPrefix` for verifying images. Read our
+previous [blog post](https://www.kubewarden.io/blog/2022/05/verifying-image-signatures/) if you want to learn more about
+how to verify container images with Sigstore using Kubewarden.
+
+## Let's see it in action!
+
+We want to verify the image `ghcr.io/raulcabello/app-example` which was signed inside a GitHub action. Let's run the
+following command to see the issuer and subject:
+
+```
+COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/raulcabello/app-example:v0.4.0
```

Comment:
If we want to keep this cli (see below) we should change it to be:

```suggestion
COSIGN_EXPERIMENTAL=1 cosign verify ghcr.io/raulcabello/app-example:v0.4.0 | jq
```

Also, don't forget to add the `console` annotation to the triple backtick